tests/integration/004_template/main.tf

```diff
@@ -31,6 +31,10 @@ resource "env0_template" "github_template" {
   terraform_version                       = "0.15.1"
 }
 
+data "env0_source_code_variables" "variables" {
```

Comment:
Not very useful, but better than nothing. I can't really create source code variables.

Reply:
You can test the outputs.

Reply:
Do you know if the repo `https://github.com/env0/templates` (which is used for integration testing) has any source variables?

Reply:
updated the integration test based on your recommendation.